modules/apps/user-groups-admin/user-groups-admin-impl/src/main/java/com/liferay/user/groups/admin/internal/search/spi/model/index/contributor/UserGroupModelDocumentContributor.java

```diff
@@ -36,6 +42,23 @@ public void contribute(Document document, UserGroup userGroup) {
 				userGroup.getUserGroupId()));
 	}
 
+	private long[] _getUserGroupGroupIds(UserGroup userGroup) {
```

Comment:
Maybe this could be changed by:

```
return TransformUtil.transformToLongArray(_groupLocalService.getUserGroupGroups(
			userGroup.getUserGroupId()),(group)-> group.getGroupId());
```